hera_mc/cm_handling.py

```diff
@@ -153,7 +153,7 @@ def __find_ports(self, connections_dict, current):
                 elif ':down:' in k and v.upstream_output_port not in output_ports:
                     output_ports.append(v.upstream_output_port)
                 else:
-                    print("cm_handling[146]: ERROR SHOULD BE UP or DOWN ", k)
+                    print("cm_handling[156]: WARNING SHOULD BE UP or DOWN ", k)
```

Comment:
After you print this warning the code will just keep on running along merrily. Is that the best behavior? Should it raise an exception instead?